install/kubernetes/cilium/values.yaml

```diff
@@ -1024,13 +1025,13 @@ hubble:
 # -- Method to use for identity allocation (`crd` or `kvstore`).
 identityAllocationMode: "crd"
 
-# TODO: Add documentation
+# -- Time to wait before using new identity on endpoint identity change (default 5s).
 # identityChangeGracePeriod: "5s"
 
-# TODO: Add documentation
+# -- GC interval for security identities (default 15m0s).
 # identityGCInterval:
 
-# TODO: Add documentation
+# -- Timeout after which identity expires on lack of heartbeat (default 30m0s).
 # identityHeartbeatTimeout: ""
```

Comment:
Why aren't those lines getting their default value assigned as for e.g. `identityChangeGracePeriod`, even if they're commented?